### DISCUSSION TITLE
Recent and Most Used tools section

### DISCUSSION THREAD

**Discussion by hasansezertasan** [dailydotdev/daily#801] [Feature Requests]
I know it is a bit off-topic but I would like to see my favorite tools on my daily.dev dashboard - yeah it's like a dashboard to me.

devdocs.io is one of my most visited pages and I frequently visit diffchecker.com, there are tools that I'm missing all the time, googling back, and sometimes I can't find them again. I know devdocs isn't just a tool and diffchecker is something else but they are mostly dev related and there are tons of applications, tools, and websites that we are missing out. Daily.dev could help us to keep them in mind!

I have a bunch of bookmark folders called "dev tools", the same name again and again. 

I loved the design of the page and the fact is that it inspires me a lot! This would be an awesome feature for me. Think about it, a developer can search through the database of the tool which was added today (five years before, from five years today)

Thank you, team.

**Comment by idoshamun** [comment]
Hi @vimevim,

Can you elaborate more on how you envision it, please?
Is it simply shortcuts that lead to your favorite tools, or what exactly are you expecting?

**Reply by hasansezertasan** [reply]
Hmm, let me try to explain it a bit detailed. 

Daily.dev has browser extensions for various of browsers. I may allow these extensions to track my activity on these tools-websites. This extension can say "your favorite tools are these" and I can skip googling or trying to remember "which tool was it?" and I can see my recent visits on these tools and can see the frequent  tools I used in my browser.

If daily.dev would have a "bookmark your tools" section, I would add them here and in time more people would add these tools.

Actually, bookmarking or saving urls would help. I used to have a repository called my-tools and I would record the urls of the tools I use daily in a markdown file. That wasn't a great practice because there are hundreds of new tools every week we have. In time, some gold gets old and we can skip them... I have a bunch of bookmark folders in my bookmark bar and their purpose is basically help me to find a tool I used once or twice and I find it useful and could use again.

And here comes, it could be an another extension or application. It's of topic but would be a great utility for me and probably for many. An application with a database of urls and dev related sites which we filter, search through it.

BTW, searching is great, I'm not complaining about it. But discovery gets hard and there are valuable sites, tools somewhere in a Google search results which is not promoted enough...

People are writing blogs titled "10 great extensions you should discover" or "amazing websites every developer should know" and which is great but they write these blogs because it attracts people. I'm sure someone will or already wrote a blog post titled similar and contains "daily.dev" as a sub title.


I hope you can see where I'm going with it. It would be helpful.

**Reply by idoshamun** [reply]
Thanks for explaining!

**Comment by merwusch** [comment]
I see that you have improved the mentioned section. I would like to offer a couple of suggestions as well. I think it would be more practical if titles could be added for the sites. Furthermore, making the buttons slightly larger could provide a more appealing UI.
